server/src/Validation/Auth.validation.js

```diff
@@ -83,6 +85,41 @@ export default class AuthValidation {
     }
     return next();
   }
+
+  /**
+   * @method validateLogin
+   *
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   *
+   * @returns {object} status code and error message properties
+   */
+  static validateLogin(req, res, next) {
+    const { email, password } = req.body;
+
+    if (isEmpty(email) && isEmpty(password)) {
+      return res.status(400).json({
+        status: 400,
+        error: 'Email and password is required'
+      });
+    }
+
+    if (isEmpty(email)) {
```

Comment:
Nice job checking that the email exists. I think it will be nice to also check if the email is a real email. What do you think?

Reply:
nice suggestion....done